app/presenters/tree_builder_vat.rb

```diff
@@ -12,7 +12,7 @@ def initialize(name, type, sandbox, build = true, root = nil, vat = nil)
   private
 
   def root_options
-    image = "100/vendor-#{@root.image_name}.png"
+    image = "svg/vendor-#{@root.image_name}.svg"
```

Comment:
we will have to rethink these `root_options` because on some places it's an image, on some place it's an icon